2.6/s2i/bin/run

```diff
@@ -0,0 +1 @@
+/bin/run-mongod
```

Comment:
yes it's related to #246, but I guess it's ok at least for now.  The entrypoint for the user-built image will be "container-entrypoint" still and the default command will be this run script, the user can change the command to "/bin/run-mongod-replication" if they want to use replication.